camera_system/coordinate_getter.py

```diff
@@ -0,0 +1,81 @@
+"""座標取得GUIツール.
+
+画像を表示し、クリックした座標を保持する
+@author miyashita64 mutotaka0426
+@note 参考: https://techacademy.jp/magazine/51035
+"""
+
+import cv2
+from typing import List
+
+
+class CoordinateGetter:
+    """画像から座標を取得するクラス.
+
+    Attributes:
+        __block_point: ブロック置き場の座標リスト
+        __base_circle: ベースサークルの座標リスト
+    """
+
+    def __init__(self, file_name: str) -> None:
+        """カメラシステムのコンストラクタ.
+
+        Args:
+            file_name (str, optional): 入力画像ファイルのパス.
+        """
+        # メンバを初期化する.
+        self.__block_point = []
+        self.__base_circle = []
+
+        # GUIツールを設定する
+        self.__img = cv2.imread(file_name)
+        self.__window_name = "Coordinate Getter"
+        # 画像を表示する.
+        self.__showImage()
+
+    def __showImage(self) -> None:
+        """画像を表示する関数."""
+        cv2.imshow(self.__window_name, self.__img)
+        cv2.setMouseCallback(self.__window_name, self.__setCoordinate)
+        cv2.waitKey(0)
+
+    def __setCoordinate(self, event, x, y, flags, params) -> None:
+        """マウス操作で取得した座標を各座標リストにセットするコールバック関数.
+
+        Args:
+            event (str, optional): イベントを示す変数.
+            x, y (int, optional): イベント発生時のマウスポインタの座標.
+            NOTE: OpenCVのコースバック関数については https://qr.paps.jp/opDWK を参照
+        """
+        # 最初の8クリックはブロック置き場の座標を取得する
+        if len(self.__block_point) < 8:
+            if event == cv2.EVENT_LBUTTONDOWN:
+                print('ブロック置き場{}：{} {}'.format(len(self.__block_point)+1, x, y))

```

Comment:
```suggestion
                print(f'ブロック置き場{len(self.__block_point)}：{x} {y}')
```
format関数ではなくf文字列のほうが見やすいと思います．

Reply:
>print文の変数展開の方法に関しては好みだと思うので変更するかどうかはおまかせします．
この辺は、コーディング規約等で統一したほうが良さそうな気がします（正確には、文字列への変数展開？）。

```suggestion
                print('ブロック置き場%d：%d %d' % (len(self.__block_point)+1, x, y))
```
ちなみに、私は`%`演算子派です（C言語と似ているため）

また、`camera_system/camera_system.py`や`camera_system/str_to_bool.py`で既に`%`演算子を利用していることからも、`%`演算子を推したいです...

Reply:
知らないからこの手法にしてただけで特にこだわりはないから、現行の手法に合わせようか

走行システムと並行して開発してるから、出来ればc言語に近い方がいいしね